tests/functional/conftest.py

```diff
@@ -0,0 +1,122 @@
+from pathlib import Path
+import pytest
+
+from ragger import Firmware
+from ragger.backend import SpeculosBackend, LedgerCommBackend, LedgerWalletBackend
+from ragger.navigator import NanoNavigator
+from ragger.utils import app_path_from_app_name
+
+
+APPS_DIRECTORY = (Path(__file__).parent.parent / "elfs").resolve()
+
+APP_NAME = "eos"
+
+BACKENDS = ["speculos", "ledgercomm", "ledgerwallet"]
+
+FIRMWARES = [Firmware('nanos', '2.1'),
+             Firmware('nanox', '2.0.2'),
+             Firmware('nanosp', '1.0.3')]
```

Comment:
It is not available on `Speculos` and therefore `Ragger` because as there is no syscall change between 1.0.3 and 1.0.4, there was no real need to add 1.0.4 support. However, adding it would definitively still brings some clarity...